developer-community/tech-talks.md

```diff
@@ -0,0 +1,240 @@
+# The full story
+
+# Speakers
+## Contact
+
+Ideally, we should start contacting speakers __2.5 months__ prior to the event.
```

Comment:
Maybe even 3 months or more? For some speakers it might be such a short notice. It happened during the frontend tech talks if I'm not mistaken. Right, @Serabe?